package-lock.json

```diff
@@ -5438,7 +5438,8 @@
     "@firebase/auth-types": {
       "version": "0.12.0",
       "resolved": "https://registry.npmjs.org/@firebase/auth-types/-/auth-types-0.12.0.tgz",
-      "integrity": "sha512-pPwaZt+SPOshK8xNoiQlK5XIrS97kFYc3Rc7xmy373QsOJ9MmqXxLaYssP5Kcds4wd2qK//amx/c+A8O2fVeZA=="
```

Comment:
@piyushkhurana2019, you don't need to push `package-lock.json` file until and unless you haven't added any new package in `package.json`. This is the file which @PiyushKalyanpy, is talking about.